README.md

```diff
@@ -27,6 +27,15 @@ Visual Studio downloads the required dependencies and compiles the Acrolinx .NET
 
 Refer to [sidebar demo] (https://github.com/acrolinx/acrolinx-sidebar-demo-dotnet/blob/master/README.md)
 
+## SDK Features
+
+1. **Document Model** - Provides [lookup](https://github.com/acrolinx/acrolinx-coding-guidance/blob/master/topics/text-lookup.md "Lookup") functionality.
+2. **Start-page** - Provides interactive way to login to Acrolinx with in-built error handling.
+3. **Logger** - Provides logging using Log4net.
+4. **Acrolinx Storage**: Applications using IE web browser control may be denied to access local storage.
+	1. SDK uses its own storage mechanism using windows registry.
+	2. **Registry path**: HKCU\Software\Acrolinx\Plugins\Storage\[KEY] **Fallback path**: HKLM\Software\Acrolinx\Plugins\Storage\[KEY]
```

Comment:
Maybe we should also add links to the classes implementing the SDK features?

Reply:
done